packages/google_maps_flutter/google_maps_flutter_ios/example/ios12/ios/RunnerTests/GoogleMapsPolylinesControllerTests.m

```diff
@@ -0,0 +1,80 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import google_maps_flutter_ios;
+@import google_maps_flutter_ios.Test;
+@import XCTest;
+@import GoogleMaps;
+
+#import <OCMock/OCMock.h>
+#import "GoogleMapPolylineController_Test.h"
+#import "PartiallyMockedMapView.h"
+
+@interface GoogleMapsPolylinesControllerTests : XCTestCase
+@end
+
+@implementation GoogleMapsPolylinesControllerTests
+
+- (FLTGoogleMapPolylineController *)setUpPolyLineControllerWithMockedMap {
```

Comment:
This needs a declaration comment, per style guide. Test helper code is still code that humans have to understand and maintain, which is what comments are to help with.